components/partials/footer.js

```diff
@@ -0,0 +1,226 @@
+import React, { useState } from "react";
+import Image from "next/image";
+import bg from "../../assets/images/footer/new-bg.svg";
+import Link from "next/link";
+import axios from "axios";
+import config from "../../config";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faFacebookF,
+  faInstagram,
+  faTwitter,
+  faMediumM,
+  faLinkedinIn,
+  faYoutube,
+} from "@fortawesome/free-brands-svg-icons";
+import { faBell, faXmark } from "@fortawesome/free-solid-svg-icons";
+
+export default function Footer({ mixpanel }) {
+  const [email, setEmail] = useState("");
+  const [alerts, setAlerts] = useState(false);
+
+  if (alerts) {
+    setTimeout(() => {
+      setAlerts(false);
+    }, 2000);
+  }
+
+  const handleIconClick = (icon) => {
+    mixpanel.track(icon);
+  };
+
+  const handleSubmit = async (event) => {
```

Comment:
Let's rename it to `handleSubscription` or any relative name which represents its purpose